application/src/main/java/org/opentripplanner/apis/transmodel/model/framework/EmissionType.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.apis.transmodel.model.framework;
+
+import graphql.Scalars;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLObjectType;
+import org.opentripplanner.model.plan.Emission;
+
+public class EmissionType {
+
+  public static GraphQLObjectType create() {
+    return GraphQLObjectType.newObject()
+      .name("Emission")
+      .description("Emission information for a trip-pattern or legs.")
```

Comment:
Here it is really the legs, not the hops, correct?

Reply:
Yes, itineraries have legs.